engine/schema/src/main/resources/META-INF/db/schema-41810to41900.sql

```diff
@@ -135,3 +135,7 @@ CREATE VIEW `cloud`.`async_job_view` AS
 UPDATE `cloud`.`console_session` SET removed=now();
 -- Modify acquired column in console_session to datetime type
 ALTER TABLE `cloud`.`console_session` DROP `acquired`, ADD `acquired` datetime COMMENT 'When the session was acquired' AFTER `host_id`;
+
+ALTER TABLE `cloud`.`kubernetes_cluster` ADD COLUMN is_managed tinyint(1) DEFAULT 1;
+
+ALTER TABLE `cloud`.`kubernetes_cluster` MODIFY COLUMN `kubernetes_version_id` bigint unsigned NULL;
```

Comment:
the field has a comment before, will the comment be removed after modification ?
```
  `kubernetes_version_id` bigint unsigned NOT NULL COMMENT 'the ID of the Kubernetes version of this Kubernetes cluster',
```